features/FooterUpdate.feature

```diff
@@ -0,0 +1,67 @@
+@owner:derek
+@author:lilla
+Feature: Docs Footer Update
+
+    New responsive design and links for the footer on the docs pages.
+
+
+    @motivating
+    Scenario: The footer displays resources with distinct links grouped into 4 different categories
+        Given the user viewing the footer
+        Then then they see the following categories and links
+
+            | Category  | Sub-category      | Link                                                                          |
+            | Product   | Login             | https://app.referralsaasquatch.com                                            |
+            | Product   | Customer Loyalty  | https://www.saasquatch.com/loyalty-program-software/                          |
+            | Product   | Customer Referral | https://www.saasquatch.com/referral-program-software/                         |
+            | Product   | Request Demo      | https://www.saasquatch.com/request-demo/                                      |
+            | Product   | Integrations      | https://www.saasquatch.com/api-integrations/                                  |
+            | Product   | Status Page       | https://status.saasquatch.com/                                                |
+            | Company   | About             | https://www.saasquatch.com/about/                                             |
+            | Company   | Partner Program   | https://www.saasquatch.com/partners/                                          |
+            | Company   | Careers           | https://www.saasquatch.com/careers/                                           |
+            | Company   | Contact Us        | https://www.saasquatch.com/contact/                                           |
+            | Resources | Blog              | https://www.saasquatch.com/blog/                                              |
+            | Resources | Podcast           | https://www.saasquatch.com/the-advocacy-channel-a-customer-marketing-podcast/ |
+            | Resources | Case Studies      | https://www.saasquatch.com/customers/                                         |
+            | Resources | Loyalty Academy   | https://www.digitalloyaltyacademy.com/                                        |
+            | Legal     | Terms of Service  | https://www.saasquatch.com/terms-of-service/                                  |
+            | Legal     | Privacy Policy    | https://www.saasquatch.com/privacy-policy/                                    |
+            | Legal     | Terms of Use      | https://www.saasquatch.com/terms-of-use/                                      |
+
+        When they click on a sub-category the link is opening up
```

Comment:
your `When` should only include the user action `they click on a sub-category link` the result needs to be in a `Then`